tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature

```diff
@@ -42,11 +30,17 @@ Feature: MATCH Engagements submissions page
     Then I should be on "/node/445/edit"
     When I press "Save"
     Then I should be on "/node/445"
-
-    #TODO: test team column displays mentor, student, consultant, and steering committee assignments
+    #TODO: test team column displays consultant
     # see https://cyberteamportal.atlassian.net/browse/D8-1593
+    When I am on "/match-engagements-submissions"
+    Then I should see "Team"
+    Then I should see "Researcher: Xiaoqin Huang"
+    Then I should see "Mentor: Xiaoqin Huang"
+    Then I should see "Steering Committee: Alana Romanella"
+    Then I should see "Students: Alexandra Lamtyugina"
+
```

Comment:
@a-pasquale -- do we want to test for a specific engagement like this?  my hunch is we should create one, and verify it looks good, yes?

Reply:
This is fine for the short term. We can revisit and create an engagement with behat or amp_dev in the future.